clueless/templates/clueless/makeSuggestion.html

```diff
@@ -1,8 +1,48 @@
 {% load static %}
-    <section id="intro" class="intro-section" style="margin-top:60px">
-        <div class="container">
-            <div class="row">
-                    <h2 class="form-suggestion-heading">What is your Suggestion?</h2>
-            </div>
-        </div>
-    </section>
+<form class="form-suggestion" action="{% url 'make_suggestion_controller' game.id player.id %}" method="post" id="suggestionTurnForm" >{%csrf_token%}
+	<input type="hidden" name="game_id" value="{{ game.id }}" />
+    <input type="hidden" name="player_id" value="{{ player.id }}" />
+    <input type="hidden" name="room_id" value="{{ room.card_id }}" />
+
+    <h2 class="form-suggestion-heading">What is your Suggestion?</h2>
+	 <br>
+    <h4>Room: {{ room.name }}</h4>
+	 <br>
+	  <label for="suggestion_suspect">Select suspect:</label>
+		<select class="form-control" name="suspect_id" id="suggestion_suspect">
+            {% for csi in characterSheetItems %}
+                <option value="{{ csi.card.card_id }}" aria-hidden="true">
+                    {% if csi.initiallyDealt %}
+                        {{ csi.card.name }} &nbsp;&nbsp;[dealt initially]
+                    {% elif csi.checked and csi.manuallyChecked %}
```

Comment:
I'd like to change [dealt initially] to like [card is in hand] or something slightly more self-explanatory.